sklearn_bench/README.md

```diff
@@ -90,6 +90,18 @@ You can launch benchmarks for each algorithm separately. The tables below list a
 | ----- | ---- |---- |---- |
 | metric | str | cosine | *cosine* or *correlation* Metric to test for pairwise distances |
 
+### HistGradientBoostingClassifier & HistGradientBoostingRegressor
+
+| parameter Name  | Type | default value | description |
+| ----- | ---- |---- |---- |
+| n-estimators | int | 100 | The number of gradient boosted trees |
+| objective | str | | *reg:squarederror*, *binary:logistic*, *multi:softprob* |
+| reg-lambda | float | 0 | L2 regularization term on weights |
+| learning-rate | float | 0.3 | Step size shrinkage used in update to prevents overfitting |
```

Comment:
```suggestion
| learning-rate | float | 0.3 | Step size shrinkage used in update to prevent overfitting |
```

`Step size shrinkage used in update` -- unclear